docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -374,21 +299,23 @@
             deploy_config: "/home/{{ ansible_ssh_user }}/deployment-config.yaml"
           when: inventory_hostname != 'localhost'
 
-        - name: Configure required region value for subcloud
+        - name: Configure required region value if not systemcontroller
```

Comment:
I'm a bit concerned about why there's a change from the existing commit? Seems from a resolve of merge conflicts. We'll need to monitor the result after the merge

Reply:
Will monitor, once the result will be merged.